_episodes/01-introduction.md

```diff
@@ -9,7 +9,42 @@ objectives:
 keypoints:
 - "First key point. Brief Answer to questions. (FIXME)"
 ---
-FIXME
+# 1. What is performance?
+
+A brief note on computer performances.
+
+Factors that affect performances:
```

Comment:
Okay, I am right now working on "learner profile" part. The rest of your suggestions on 01-introduction.md is duly noted, but the materials included in these episodes were initial ideas that are subject to change, and I think with time these episodes would be completely eliminated and replaced with new contents. So I am not focussing on this right now. Important part here is to finalise the skeleton of the skeleton of the course and then working on each segment separately. So, it would be great to have a schedule something like this: https://hpc-carpentry.github.io/hpc-intro/